studio/pages/project/[ref]/settings/database.tsx

```diff
@@ -303,14 +303,18 @@ const GeneralSettings: FC<any> = ({ projectRef }) => {
   const DB_FIELDS = ['db_host', 'db_name', 'db_port', 'db_user', 'inserted_at']
   const connectionInfo = pluckObjectFields(formModel, DB_FIELDS)
 
-  const defaultConnString =
+  const defaultURIConnString =
     `postgresql://${connectionInfo.db_user}:[YOUR-PASSWORD]@` +
     `${connectionInfo.db_host}:${connectionInfo.db_port.toString()}` +
     `/${connectionInfo.db_name}`
   const golangConnString =
     `user=${connectionInfo.db_user} password=[YOUR-PASSWORD] ` +
     `host=${connectionInfo.db_host} port=${connectionInfo.db_port.toString()}` +
     ` dbname=${connectionInfo.db_name}`
+  const defaultPSQLConnString = 
+    `psql -h db.` + `${connectionInfo.db_host}` + `.supabase.co -p ` +
+    `${connectionInfo.db_port.toString()}` + ` -d ${connectionInfo.db_name}` +
+    ` -U ${connectionInfo.db_user}`
```

Comment:
should we make this one 

```
`psql -h db.${connectionInfo.db_host}.supabase.co -p ` +
`${connectionInfo.db_port.toString()} -d ${connectionInfo.db_name} ` +
`-U ${connectionInfo.db_user}`
```

A bit more readable!

Reply:
yes this one is more readable, thanks